integration/client/client_unix_test.go

```diff
@@ -32,7 +32,7 @@ const (
 )
 
 var (
-	testImage    = "mirror.gcr.io/library/busybox:1.32"
+	testImage    = "mirror.gcr.io/library/busybox:latest"
```

Comment:
We don't use latest here. Please update it with specific version. Thanks

Reply:
I was going to suggest the same; let's switch to `1.33` for this PR and then long-term determine how we are going to deal with mirrors / GHCR / build our own to not be caught by images coming and going

Reply:
maybe when we build integration get the list and substitute busybox:??? with the latest in the mirror... and agree we should get the bucket back up by moving to 1.33 for now

Reply:
fixed